### PR TITLE
Feature EPL-1470: Add Taxes, 303 and 390 fixes to migration doc

### PR DESCRIPTION
**Pull request by RubenEtendo**

### CODE REVIEW

File: docs/developer-guide/etendo-classic/bundles/spain-localization/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    2. Instalar dataset del modelo 303 a nivel de organizaciones individuales. Se insertarán en los impuestos de la organización * los parámetros correspondientes, asociados a cada organización específica
+
+
+??? success "EE-856 Solved"
+
+    ### [EE-856](https://etendoproject.atlassian.net/browse/EE-856){target="\_blank"} Dataset de Declaraciones de Impuestos modelo 390 duplica datos para el 2022 cuando se aplica a un servidor migrado de Openbravo a Etendo después del 01-2023.
+
+    ### Workaround
+
+    !!! warning
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/bundles/spain-localization/known-issues.md:67]: @RubenEtendo Block should not be added inside other block

File: docs/developer-guide/etendo-classic/bundles/spain-localization/known-issues.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Known Issues
+
+??? success "EE-808 Solved"
+
+    ### [EE-808](https://etendoproject.atlassian.net/browse/EE-808){target="\_blank"} Problema al intentar importar el dataset 'taxes: configuration for Spain' si el entorno ya tiene aplicado el dataset de impuestos temporales 303 de Openbravo.
+
+    #### Workaround
+
+    !!! warning
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/bundles/spain-localization/known-issues.md:20]: @RubenEtendo Block should not be added inside other block

**Review by valeg-etendo — APPROVED** [review]

File: docs/developer-guide/etendo-classic/getting-started/migration-from-openbravo/migrating-to-etendo-from-openbravo.md [hunk 3]
```diff
@@ -376,6 +376,27 @@ You may change the current user from Openbravo to Etendo, if you want. This guid
...
+### Apply 390 Taxes and Tax Report Fixes
+
+This step is only necessary if the following conditions happen in your environment:
+
+* The `org.openbravo.module.aeat303.temporal.taxes.es`, `org.openbravo.module.aeat303.temporal.taxparameters.es`, `org.openbravo.localization.spain.referencedata.taxes.es` and `org.openbravo.module.aeat390.es` modules are installed and their datasets applied in the ERP
+* You want to update the `Spain Localization Extensions` bundle to version `1.8.0` or higher, otherwise, you want to update the afforementioned modules to these versions:
+    * `org.openbravo.localization.spain.referencedata.taxes.es`: version `1.8.0` or higher
+    * `org.openbravo.module.aeat390.es`: version `3.9.0` or higher
+
+To solve issues with taxes arising from these modules being migrated from Openbravo to Etendo, refer to the related [Spain Localization Known Issue](../../../../whats-new/release-notes/etendo-classic/known-issues.md/#ee-856-dataset-de-declaraciones-de-impuestos-modelo-390-duplica-datos-para-el-2022-cuando-se-aplica-a-un-servidor-migrado-de-openbravo-a-etendo-despues-del-01-2023)
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/getting-started/migration-from-openbravo/migrating-to-etendo-from-openbravo.md:399]:
```suggestion
To solve issues with taxes arising from these modules being migrated from Openbravo to Etendo, refer to the related [Known Issue](../../../../whats-new/release-notes/etendo-classic/known-issues.md/#ee-856-dataset-de-declaraciones-de-impuestos-modelo-390-duplica-datos-para-el-2022-cuando-se-aplica-a-un-servidor-migrado-de-openbravo-a-etendo-despues-del-01-2023) 
```

File: docs/developer-guide/etendo-classic/getting-started/migration-from-openbravo/migrating-to-etendo-from-openbravo.md [hunk 4]
```diff
@@ -376,6 +376,27 @@ You may change the current user from Openbravo to Etendo, if you want. This guid
...
+### Apply 303 Taxes and Tax Report Fixes
+
+This step is only necessary if the following conditions happen in your environment:
+
+* The `org.openbravo.module.aeat303.temporal.taxes.es`, `org.openbravo.module.aeat303.temporal.taxparameters.es`, `org.openbravo.localization.spain.referencedata.taxes.es` and `org.openbravo.module.aeat303.es` modules are installed and their datasets applied in the ERP
+* You want to update the `Spain Localization Extensions` bundle to version `1.8.0` or higher, otherwise, you want to update the afforementioned modules to these versions:
+    * `org.openbravo.localization.spain.referencedata.taxes.es`: version `1.8.0` or higher
+    * `org.openbravo.module.aeat303.es`: version `1.14.0` or higher
+
+To solve issues with taxes arising from these modules being migrated from Openbravo to Etendo, refer to the related [Spain Localization Known Issue](../../../../whats-new/release-notes/etendo-classic/known-issues.md/#ee-808-problema-al-intentar-importar-el-dataset-taxes-configuration-for-spain-si-el-entorno-ya-tiene-aplicado-el-dataset-de-impuestos-temporales-303-de-openbravo)
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/getting-started/migration-from-openbravo/migrating-to-etendo-from-openbravo.md:388]:
```suggestion
To solve issues with taxes arising from these modules being migrated from Openbravo to Etendo, refer to the related [Known Issue](../../../../whats-new/release-notes/etendo-classic/known-issues.md/#ee-808-problema-al-intentar-importar-el-dataset-taxes-configuration-for-spain-si-el-entorno-ya-tiene-aplicado-el-dataset-de-impuestos-temporales-303-de-openbravo) 
```

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

**Review by valeg-etendo — APPROVED** [review]

**Review by isaiasb-etendo — APPROVED** [review]

**Review by Matias-Bernal — CHANGES_REQUESTED** [review]

File: docs/whats-new/release-notes/etendo-classic/known-issues.md [hunk 5]
```diff
@@ -26,6 +26,108 @@ This page displays the known issues reported by the support team.
 
     This workaround is applicable if you have deployed Etendo from the ISO in versions prior to [24.1.4](https://etendo-appliances.s3.eu-west-1.amazonaws.com/etendo/iso/etendo-24Q1.4.iso). Starting from that version, the issue of permission changes on the gradlew file is no longer observed.
 
+??? success "EE-856 Solved"
```
[the remark this answers is not in the document]
Comment (RubenEtendo) [reply docs/whats-new/release-notes/etendo-classic/known-issues.md:29]: Done!

File: docs/whats-new/release-notes/etendo-classic/known-issues.md [hunk 6]
```diff
@@ -26,6 +26,112 @@ This page displays the known issues reported by the support team.
...
+
+        3. Apply the Taxes dataset update for the corresponding organization.
+
+        4. Apply the dataset update for the 390 tax model.
+
+??? success "EE-808 Solved"
+
+    ### [EE-808](https://github.com/etendosoftware/com.etendoerp.localization.spain.extensions/issues/33){target="\_blank"} Problem when trying to import the 'taxes: configuration for Spain' dataset if the environment already has imported the dataset related to the 303 temporary taxes of Openbravo.
+
+    #### Workaround
```
Comment (valeg-etendo) [inline docs/whats-new/release-notes/etendo-classic/known-issues.md:88]:
```suggestion
```

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

**Review by Matias-Bernal — APPROVED** [review]

File: docs/whats-new/release-notes/etendo-classic/known-issues.md (hunk 6 above)
Comment (RubenEtendo) [reply docs/whats-new/release-notes/etendo-classic/known-issues.md:88]: Done!

**Review by valeg-etendo — APPROVED** [review]

**Pull request merged**